book/text.md

```diff
@@ -21,8 +22,8 @@ What is a Font?
 ===============
 
 So far, we've called `create_text` with a character and two
-coordinates to write text to the screen. But we never specified the
-font\index{font}, the size, or the color. To talk about those things,
+coordinates to write text to the screen. But we never specified its
+font\index{font} or size. To talk about those things,
```

Comment:
So I'm not thrilled with my change by my problem with the original text is that you don't need font objects to change the color. Maybe a better fix is just changing "the color" to "the text style"?